helm/cluster-cloud-director/values.schema.json

```diff
@@ -120,64 +184,11 @@
                 }
             }
         },
-        "network": {
-            "type": "object",
-            "properties": {
-                "cidrBlocks": {
-                    "type": "array",
-                    "items": {
-                        "type": "string"
-                    }
-                },
-                "podsCidr": {
-                    "type": "array",
-                    "items": {
-                        "type": "string"
-                    }
-                }
-            }
+        "nodeClasses": {
```

Comment:
can you add "items" section here? See https://github.com/giantswarm/cluster-openstack/blob/main/helm/cluster-openstack/values.schema.json#L146